db/migrate/20150331170704_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :rating, null: false
+      t.text :description, null: false
+      t.integer :user_id, null: false
+      t.integer :beer_id, null: false
```

Comment:
Add indexes on these foreign keys to speed up queries for reviews by user or reviews by beer.